RPLCD/common.py

```diff
@@ -84,6 +88,25 @@
 PIN_READ_WRITE = 0x2
 PIN_REGISTER_SELECT = 0x1
 
+# MCP23008 pin bitmasks
+MCP23008_RS = 0x02
+MCP23008_E = 0x4
+MCP23008_DATAMASK = 0x78
+MCP23008_DATASHIFT = 3
+
+# MCP23008 Register addresses
+MCP23008_IODIR = 0x00
+MCP23008_IPOL = 0x01
+MCP23008_GPINTEN = 0x02
+MCP23008_DEFVAL = 0x03
+MCP23008_INTCON = 0x04
+MCP23008_IOCON = 0x05
+MCP23008_GPPU = 0x06
+MCP23008_INTF = 0x07
+MCP23008_INTCAP = 0x08  # (Read-only)
+MCP23008_GPIO = 0x09
+MCP23008_OLAT = 0x0A
+
```

Comment:
You don't have to do a new pull request, you can use "git push -f" to overwrite the old branch.